codespell_lib/data/dictionary.txt

```diff
@@ -23925,11 +23935,42 @@ nanoseonds->nanoseconds
 Naploeon->Napoleon
 Napolean->Napoleon
 Napoleonian->Napoleonic
+napom->napalm
+napomed->napalmed
+napomes->napalms
+napoming->napalming
+napommed->napalmed
+napommes->napalms
+napomming->napalming
+napomms->napalms
+napoms->napalms
+narl->gnarl
```

Comment:
For `narl` and friends, add possible typos in addition to phonetic spelling:
https://dictionary.cambridge.org/spellcheck/english/?q=narl

Reply:
Added.